shacl/dataset.jsonld

```diff
@@ -70,13 +79,25 @@
           "sh:path": {
             "@id": "schema:potentialAction"
           },
-          "sh:minCount": 1,
-          "sh:class": {
-            "@id": "schema:SearchAction"
+          "sh:qualifiedValueShape": {
```

Comment:
This is the trick: because we want to differentiate between object values (`schema:FindAction` versus `schema:SearchAction`) of the same predicate (`schema:potentialAction`), we need to use qualified value shapes.

Reply:
Perfect! Thank you!